Packs/SafeBreach/Integrations/SafeBreach_v2/SafeBreach_v2.yml

```diff
@@ -300,7 +300,7 @@ script:
       description: Array of insight IDs to rerun.
       isArray: true
       name: insightIds
-      required: false
```

Comment:
Please revert this since this parameter could have been configured in the customer instance and could break his flows.
instead, you can add in the "main" section a check if the parameter exists and if not raise an exception.